packages/likert/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "@pie-element/likert",
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": "pie-framework/pie-elements",
+  "version": "0.0.1",
+  "description": "",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "dependencies": {
+    "@pie-ui/likert": "^0.0.1"
```

Comment:
hi @relumesaros - can you bump the dependency here? to `^0.1.2` - i'll then merge - thanks

Reply:
solved :)

Reply:
@edeustace this is gonna be published automatically? :)

Reply:
once I merge to master yes - but I'll need to coordinate that